microsoft-edge/webview2/samples/webview2apissample.md

```diff
@@ -1,133 +1,142 @@
 
-1. In a separate window or tab, read the rendered README.md file for this project at GitHub: [README file for WebView2APISample](https://github.com/MicrosoftEdge/WebView2Samples/tree/main/SampleApps/WebView2APISample#readme).  Then return to this page and continue the steps below.
+This sample and its solution file are unique: it contains a copy of other samples, in Solution Explorer.
 
-   * [README > Prerequisites](https://github.com/MicrosoftEdge/WebView2Samples/tree/main/SampleApps/WebView2APISample#prerequisites)
+**WebView2APISample** is a hybrid application built with the Microsoft Edge WebView2 control; that is, this app combines a native side and a browser web app side.  See [Hybrid app approach](../index.md#hybrid-app-approach) in _Introduction to Microsoft Edge WebView2_.
 
-   * [README > Build the WebView2 API Sample](https://github.com/MicrosoftEdge/WebView2Samples/tree/main/SampleApps/WebView2APISample#build-the-webview2-api-sample)
+The running **WebView2APISample** app window shows the WebView2 SDK version and also the WebView2 Runtime version and path.  There are many useful menus and menuitems provided for you:
 
-   You can also view the README.md source file (non-rendered) in Visual Studio.  In **File Manager** or Visual Studio > Solution Explorer, open the file:<!-- todo: is there a .md preview capability locally? -->
-
-   `<your-repos-directory>/WebView2Samples/SampleApps/README.md`
+![WebView2APISample app window showing WebView2 SDK version and WebView2 Runtime version and path](./webview2apissample-images/webview2apisample-app-window.png)
+<!-- old:
+![Sample App Snapshot](https://raw.githubusercontent.com/MicrosoftEdge/WebView2Samples/master/SampleApps/WebView2APISample/documentation/screenshots/sample-app-screenshot.png)
+-->
 
-   or:
+If this is your first time using WebView, we recommend first following the tutorial [Get started with WebView2 in Win32 apps](../get-started/win32.md), which goes over how to create a WebView2 app and walks through some basic WebView2 functionality.  That particular tutorial doesn't start with you creating a new Win32 project using a project template; instead, it starts with a finished project in the WebView2Samples repo, and walks you through how to optionally re-add the WebView2 code.
+<!-- the getstart tut should probably create new app from project template, but, currently starts from existing project, that is provided now as a finished project -->
 
-   `<your-repos-directory>/WebView2Samples-main/SampleApps/README.md`
+For details of events and API handlers in WebView2, see [WebView2 API Reference](../webview2-api-reference.md).
 
 
 <!-- ====================================================================== -->
-## Step 2 - Install Visual Studio
+## Step 1 - Prerequisite: Install a preview channel of Microsoft Edge
```

Comment:
Let's either do Step 1, step 2, ... or Prerequisites and then step 1, step 2, ... instead of both

Reply:
removed "Prerequisite: " globally in these articles h2s